libfwupdplugin/meson.build

```diff
@@ -37,11 +37,14 @@ fwupdplugin_src = [
 ]
 
 if host_machine.system() == 'linux'
-  fwupdplugin_src += 'fu-common-linux.c'  # fuzzing
+  fwupdplugin_src += 'fu-common-linux.c'    # fuzzing
+  fwupdplugin_src += 'fu-efivar-linux.c'
 elif host_machine.system() == 'freebsd'
   fwupdplugin_src += 'fu-common-freebsd.c'
+  fwupdplugin_src += 'fu-efivar-freebsd.c'
 elif host_machine.system() == 'windows'
   fwupdplugin_src += 'fu-common-windows.c'
+  fwupdplugin_src += 'fu-efivar-windows.c'  # fuzzing
```

Comment:
by splitting this way I don't think this code will ever be fuzzed

Reply:
That was intentional; i.e. we don't actually link giounix into the fuzzer binary. We certainly _could_, but the efivar parsing isn't something that lends itself to fuzzing.